cl/cleanup/management/commands/fix_tax_court.py

```diff
@@ -0,0 +1,147 @@
+# !/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import re
+
+from cl.citations import find_citations
+from cl.lib.command_utils import VerboseCommand, logger
+from cl.search.models import Opinion, OpinionCluster, Docket, Citation
+
+
+def get_tax_docket_numbers(opinion_text):
+    """
+    get_docket_numbers parses the opinon plain text for docket numbers.
+
+    First we idenitify where the docket numbers are in the document.
+    This is normally at the start of the document but can often follow
+     a lengthy case details section.
+
+    :param opinion_text: is the opinions plain_text
+    :return docket_string: as string of docket numbers Ex. (18710-94, 12321-95)
+    """
+    parsed_text = None
+    regex = r"Docket No.*.Filed|Docket No.*.(, [0-9]{4}.)"
+    matches = re.finditer(regex, opinion_text)
+    r = r"[0-9]{3,5}-[0-9A-Za-z]{2,4}(\.)|([0-9]{3,5}-[0-9A-Za-z]{2," \
```

Comment:
I think this can be: `r = r"[0-9]{3,5}-[0-9A-Za-z]{2,4}(\.)( [A-Z](\.) )?"`

Reply:
```r = r"[0-9]{3,5}-[\w]{2,4}(\.)( [A-Z](\.))?"```
I like this even better, and I dont think having _ in ```\w``` will cause any issues.